### ISSUE TITLE
Handle clicking outside the table

### ISSUE THREAD

**Issue by haxxmaxx** [qlik-oss/sn-table#135]
When loosing focus from the table by clicking somewhere else, it should remove tabindex. This can be done with a listener on the document, looking if target has a parent inside the table. Possibly also send something to nebuala via the useAccessibility hook.

**Comment by haxxmaxx** [comment]
Requirements:
- When clicking outside a table, the tabindex should be removed from the previously focused cell.
